src/Query/User/Model/PackageDetails.php

```diff
@@ -11,6 +11,7 @@ final class PackageDetails
     private string $id;
     private string $organizationId;
     private ?string $name;
+    private ?string $url;
```

Comment:
I've made it nullable as the `name` but seems it can not be null in the model
should I change it?
also what about making `name` not nullable with a fallback to the url if not present when flushing?

Reply:
Package url can't be null (when you add package to repman, you must provide url). The name is another matter. It is extracted on sync, which may fail and therefore may be null.

Reply:
yes I have understood this
I was asking to, as fallback to avoid null, define name as url by default, and the sync may/will update it :)

Reply:
we would prefer name to remain null if it is not present :wink:

Reply:
I understood